src/status_im2/common/log.cljs

```diff
@@ -37,10 +38,16 @@
                                 string/lower-case
                                 keyword))
         (log/merge-config!
-         {:output-fn (fn [& data]
-                       (let [res (apply log/default-output-fn data)]
-                         (add-log-entry res)
-                         res))})
+         {:output-fn  (fn [& data]
+                        (let [res (apply log/default-output-fn data)]
+                          (add-log-entry res)
+                          res))
+          :middleware [(fn [data]
```

Comment:
I've added a bit of formatting to printed logs for better readability. It is more convenient in development, but not sure if can cause any negative side effects. Please tell me if so!

Reply:
maybe worth sharing in the mobile channel? 🤷‍♂️